class-tgm-plugin-activation.php

```diff
@@ -439,7 +439,7 @@ public function install_plugins_page() {
 				<?php $plugin_table->prepare_items(); ?>
 
 				<?php
-				if ( isset( $this->message ) ) {
+				if ( isset( $this->message ) && is_string( $this->message ) && '' !== $this->message ) {
```

Comment:
Per the other discussion, I think the last part here should use `empty()`.

Alternatively, as things stand, the second part is redundant, since the last part is already checking the type is a string.

So, either:

~~~php
if ( isset( $this->message ) && is_string( $this->message ) && ! empty( $this->message ) ) {
~~~

or

~~~php
if ( isset( $this->message ) && '' !== ( $this->message ) ) {
~~~

should be sufficient.